package.json

```diff
@@ -402,6 +402,6 @@
   "dependencies": {
     "@types/plist": "^3.0.2",
     "plist": "^3.0.5",
-    "vscode-languageclient": "^8.0.0"
+    "vscode-languageclient": "^8.0.1"
```

Comment:
Any reason to force 8.0.1?

Reply:
No specific reason, this was in my previous patch, but there shouldn't be anything specific to 8.0.1 here.